masonry/src/passes/compose.rs

```diff
@@ -75,6 +75,10 @@ fn compose_widget(
 pub(crate) fn run_compose_pass(root: &mut RenderRoot) {
     let _span = info_span!("compose").entered();
 
+    if root.root_state().needs_compose {
+        root.global_state.pointer_changed = true;
```

Comment:
A comment may be useful here. Is this because the composition changed, so the relative location of the pointer may have changed?